lab2/src/main/java/agh/ics/oop/OptionsParser.java

```diff
@@ -0,0 +1,18 @@
+package agh.ics.oop;
+
+public class OptionsParser {
+     public static MoveDirection[] parse(String[] tab){
+       MoveDirection[] moveTab = new MoveDirection[tab.length];
+        int j=0;
+
+        for (int i = 0; i < tab.length; i++) {
+            switch (tab[i]){
+                case "f"->moveTab[j++] = MoveDirection.FORWARD;
+                case  "b"->moveTab[j++] = MoveDirection.BACKWARD;
+                case  "l"->moveTab[j++] = MoveDirection.LEFT;
+                case  "r"->moveTab[j++] = MoveDirection.RIGHT;
+            }
+        }
+        return moveTab;
```

Comment:
brakuje mechanizmu pomijania nieprawidłowych pozycji - jeśli takowe się pojawią to tablica będzie miała na końcu nulle.